Dockerfile

```
@@ -101,7 +180,7 @@ RUN curl -sSL "https://dl.dropbox.com/s/gwf51ykkk5bifyj/ants-Linux-centos6_x86_6
     | tar -xzC $ANTSPATH --strip-components 1
 
 # Installing SVGO and bids-validator
-RUN npm install -g svgo bids-validator@1.5.4 \
+RUN npm install -g svgo@^2.3 bids-validator@^1.7 \
```

Reusing cache, so not detecting that it should update this.

```suggestion
RUN npm install -g svgo@^2.3 bids-validator@1.7.2 \
```

---

let's try it